docs/guides/hooks-list.rst

```diff
@@ -585,6 +585,14 @@ Other
 	"container_guid" (int) are provided. An empty entity value generally means the form is to
 	create a new object.
 
+**elgg_channels:send, <channel_name>**
```

Comment:
I imagined a plugin's JS module would base listening on particular data- attrs from the HTML or it would listen to all and filter name via regex. Any other ideas? Maybe each message has free-form tags and you can filter via some tag-matching expression?